packages/lesswrong/components/messaging/FriendlyInbox.tsx

```diff
@@ -227,16 +231,21 @@ const FriendlyInbox = ({
   const { document: fetchedSelectedConversation } = useSingle({
     documentId: conversationId,
     collectionName: "Conversations",
-    fragmentName: "ConversationsList",
+    fragmentName: "ConversationsListWithReadStatus",
     skip: !conversationId,
   });
   const selectedConversation = fetchedSelectedConversation || eagerSelectedConversation;
 
+  const onOpenConversation = useCallback(async (conversationId: string) => {
+    await markConversationRead(conversationId);
+    await refetchConversations();
```

Comment:
Refetching is a little bit janky, but it'll do for now. We can think about how to make this smoother in the future.